app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -73,14 +71,7 @@ function UnknownError({
     );
   }
   if (isFailedDocType && err) {
-    return (
-      <p key={`${err.message}-${remainingSubmitAttempts}`}>
-        {err.message}{' '}
-        <HtmlTextWithStrongNoWrap
-          text={t('idv.warning.attempts_html', { count: remainingSubmitAttempts })}
-        />
-      </p>
-    );
+    return <p key={`${err.message}`}>{err.message} </p>;
```

Comment:
1. We could simplify to remove template string, since we're no longer concatenating?
2. Looks like an extra space before the closing tag snuck in and could be removed

```suggestion
    return <p key={err.message}>{err.message}</p>;
```

Reply:
Thanks for catching this!

A vestige from when this `return` combined two messages and I had meant to fix it and forgot. Glad you caught it! Fixed in [a836997](https://github.com/18F/identity-idp/pull/10635/commits/a8369972af17edb9be8a7d4b3a07b9d02e0130d4).